electrum/gui/qml/components/InvoiceDialog.qml

```diff
@@ -223,12 +223,13 @@ ElDialog {
                                 Layout.preferredWidth: amountFontMetrics.advanceWidth('0') * 14 + leftPadding + rightPadding
                                 fiatfield: amountFiat
                                 readOnly: amountMax.checked
+                                msatPrecision: invoice.invoiceType == Invoice.LightningInvoice
```

Comment:
The web ui does not let me comment on the line I actually want to:
https://github.com/spesmilo/electrum/blob/4a63916d5ee28392f74f2bcac5b3c27f8a51b59c/electrum/gui/qml/components/InvoiceDialog.qml#L184

Shouldn't that show text at max precision? e.g. `text: Config.formatMilliSats(invoice.amount, false)`

As-is, if you scan an invoice that has msat precision, we currently don't show the user the msat part, but we do pay it if they click pay. It feels wrong to send out more money than what we display beforehand.
but it's only sub-satoshi stuff, so ultimately we could ignore it